### PR TITLE
DOCSP-40655 Update Linux .dep installation instructions

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Update Linux .dep installation instructions to modern approaches

## STAGING

https://preview-mongodbltranmdb2.gatsbyjs.io/compass/DOCSP-40655/install/#download-and-install-compass-1

## JIRA

https://jira.mongodb.org/browse/DOCSP-40655

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=667c776febce19f314617eac

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/install.txt [hunk 1]
```diff
@@ -172,7 +172,17 @@ To download and install |compass|, select your operating system:
 
                .. code-block:: shell
 
-                  sudo dpkg -i mongodb-compass_{+current-version+}_amd64.deb
+                  sudo apt install mongodb-compass_{+current-version+}_amd64.deb
+                  
+               .. note::
+
+                  If your linux distribution does not support ``apt``,
```
Comment (sarah-olson-mongodb) [inline source/install.txt:179]: (proper noun)
```suggestion
                  If your Linux distribution does not support ``apt``,
```

File: source/install.txt [hunk 2]
```diff
@@ -172,7 +172,17 @@ To download and install |compass|, select your operating system:
 
                .. code-block:: shell
 
-                  sudo dpkg -i mongodb-compass_{+current-version+}_amd64.deb
+                  sudo apt install mongodb-compass_{+current-version+}_amd64.deb
+                  
+               .. note::
```
Comment (sarah-olson-mongodb) [inline source/install.txt:177]: (optional suggestion) 
This is subjective. I think this page might be getting a little note heavy. What do you think about making it just a second paragraph?

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @ltran-mdb2 LGTM with a couple minor nits. 
One other thing. I know this isn't your writing. But all the steps on that page could use punctuation. Would you mind adding as part of this?

File: source/install.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/install.txt]: Done!

File: source/install.txt (hunk 2 above)
Comment (ltran-mdb2) [reply source/install.txt]: Done! I also added punctuation to the steps on this page, thanks for your feedback!

File: source/install.txt [hunk 3]
```diff
@@ -161,20 +161,28 @@ To download and install |compass|, select your operating system:
...
 
                   wget https://downloads.mongodb.com/compass/mongodb-compass_{+current-version+}_amd64.deb
                
 
-            #. Install |compass|
+            #. Install |compass|.
+
+               .. code-block:: shell
+
+                  sudo apt install mongodb-compass_{+current-version+}_amd64.deb
```
Comment (addaleax) [inline source/install.txt:175]:
```suggestion
                  sudo apt install ./mongodb-compass_{+current-version+}_amd64.deb
```

`apt` needs a relative path for installing `.deb` files

**Review by gribnoysup — APPROVED** [review]

File: source/install.txt [hunk 4]
```diff
@@ -161,20 +161,28 @@ To download and install |compass|, select your operating system:
...
+
+               If your Linux distribution does not support ``apt``,
```
Comment (addaleax) [inline source/install.txt:177]: All relevant Linux distros that support `dpkg` also support `apt`. This should probably say something along the lines of

```suggestion
               If your Linux distribution does not support installing local ``.deb`` files through ``apt``,
```

File: source/install.txt (hunk 3 above)
Comment (ltran-mdb2) [reply source/install.txt]: Done!

File: source/install.txt (hunk 4 above)
Comment (ltran-mdb2) [reply source/install.txt]: Done!

**Review by addaleax — APPROVED** [review]

**Pull request merged**